api/src/paths/project/{projectId}/survey/{surveyId}/attachments/upload.ts

```diff
@@ -74,7 +64,7 @@ POST.apiDoc = {
             },
             attachmentType: {
               type: 'string',
-              enum: ['Report', 'Other']
+              enum: ['Other']
             },
             attachmentMeta: {
```

Comment:
attachmentMeta can also be removed from this endpoint, since regular attachments dont have meta